client/assets/stylesheets/shared/_reset.scss

```diff
@@ -130,7 +130,3 @@ textarea {
 	border-radius: 0;
 	appearance: none;
 }
-input[type='radio'],
-input[type='checkbox'] {
```

Comment:
Not sure if having this rule in `input` is enough or was there some reason that therese were applied to radio and checkbox specifically.

Needs testing in Safari.